umbraco/Umbraco-CMS
# Add health checks results to payload

### Prerequisites


### Description
Update the `HealthCheckCompletedWebhookEvent`. Add the health check results to the payload. 

Before: 
![image](https://github.com/umbraco/Umbraco-CMS/assets/7831614/6632042e-83ed-4e10-98e8-5b8fc25fad50)

After: 
![image](https://github.com/umbraco/Umbraco-CMS/assets/7831614/c62ba0b7-54f9-4abc-94aa-d27fd7525e98)


Test: 

- Add a composer 
```csharp
public class WebhookComposer : IComposer
{
    public void Compose(IUmbracoBuilder builder)
    {
        builder.WebhookEvents().AddCms();
    }
}
```
- Configure a Health Check Completed webhook: 

![image](https://github.com/umbraco/Umbraco-CMS/assets/7831614/2e34dffe-b075-47fd-8e30-a7bf9eb8554c)

- Run one or multiple health checks
